ynr/apps/candidates/tests/test_models.py

```diff
@@ -104,3 +151,85 @@ def test_has_results_and_no_results(self):
         all_winners = set(has_results + has_winner_no_result)
         self.assertEqual(set(Ballot.objects.has_results()), all_winners)
         self.assertEqual(set(Ballot.objects.no_results()), set(no_results))
+
+    def test_uncontested_when_winner_count_same_as_memberships(self):
+        parties = self.create_parties(2)
+        memberships = self.create_memberships(self.ballot, parties)
+        self.assertTrue(self.ballot.uncontested)
```

Comment:
Actually looking again these tests should be on the `TestBallotMethods` class on line 55 - as they are testing the ballot instance, rather than the BallotQueryset